routing/fake_edges_container.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "routing/index_graph_starter.hpp"
+
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <utility>
+
+namespace routing
+{
+class FakeEdgesContainer final
+{
+  friend class IndexGraphStarter;
+
+public:
+  FakeEdgesContainer(IndexGraphStarter && starter)
+    : m_finishId(starter.m_finishId), m_fake(std::move(starter.m_fake))
+  {
+  }
+
+  explicit FakeEdgesContainer(unique_ptr<IndexGraphStarter> starter)
```

Comment:
Is it always guaranteed that starter is not null? Why do you need such strange ctor? Why not to:
```c++
unique_ptr<IndexGraphStarter> starter = make_unique(...);
...
// Here we sure that starter is non-null
FakeEdgesContainer container(move(*starter));
```

Reply:
I mean that you may re-use the first ctor.

Reply:
ok, done